airbyte-integrations/connectors/source-linkedin-ads/source_linkedin_ads/source.py

```diff
@@ -336,6 +336,59 @@ class AdCreativeAnalytics(LinkedInAdsAnalyticsStream):
     pivot_by = "CREATIVE"
 
 
+class AdCreativeName(LinkedinAdsStream):
+
+    endpoint = "adCreativesV2"
+    search_param = "projection"
+    search_param_value = "(variables(data(*,com.linkedin.ads.SponsoredUpdateCreativeVariables(*,share~(subject,text(text),content(contentEntities(*(description,entityLocation,title))))))))"
```

Comment:
What exactly are we looking for using this parameter? Can we simplify this? 
Do we update something with this part:
`..com.linkedin.ads.SponsoredUpdateCreativeVariables..` ?

Reply:
In documentation this parameter requires such value.
[Here](https://learn.microsoft.com/en-us/linkedin/marketing/integrations/ads/account-structure/create-and-manage-creatives?tabs=curl&view=li-lms-unversioned)

Reply:
Didn't manage to find what you've meant, probably the value is comprehensive, please take a look again on it, probably we can unify/sumplify this value parameter based on Creatives stream logic

Reply:
[Here](https://learn.microsoft.com/en-us/linkedin/marketing/integrations/ads/account-structure/create-and-manage-creatives?tabs=curl&view=li-lms-unversioned#get-a-creative-name)